tools/ovc/openvino/tools/ovc/cli_parser.py

```diff
@@ -395,7 +395,9 @@ def get_common_cli_parser(parser: argparse.ArgumentParser = None):
 
     # Command line tool specific params
     parser.add_argument('--output_model',
-                              help='This parameter is used to name output .xml/.bin files with converted model.')
+                              help='This parameter is used to name output .xml/.bin files of converted model. '
+                                   'Model name or output directory can be passed. If output directory is passed, '
+                                   'the resulting .xml/.bin files are named by original model name.')
```

Comment:
For me it looks overcomplicating option. I think we should not treat `output_model` as a directory and we should always treat it as a output model name. If extension `.xml` is not in `output_model`, let us add it. If user specifies separator, let us raise exception that it should not be a directory.

Reply:
@slyalin, please advice. I think it should be used as for directory naming. What is your thought?

Reply:
But it works like this already now, in the released version of `ovc`, this is not something new. @popovaan is just providing more complete documentation for this option, describing its current behavior. And this is actually useful and expected from the user side, because we have the original model name and can reuse it. In contrast to `save_model` it makes sense.